### ISSUE TITLE
Trastuzumab Emtansine incorrectly mapped to Trastuzumab

### ISSUE THREAD

**Issue by abedtashh** [OHDSI/Vocabulary-v5.0#169]
## Probem:
"Trastuzumab" and "Trastuzumab Emtansine" are two different ingredients, but Trastuzumab Emtansine in GRR, BDPM, and dm+d vocabularies are mapped to RxNorm concept of Trastuzumab (Concept ID: 224905):

RELATES TO|CONCEPT ID|VOCABULARY
-- | -- | --
Trastuzumab Emtansine | 41605044 | GRR
Trastuzumab emtansine | 43243659 | BDPM
Trastuzumab emtansine | 21309257 | dm+d
Trastuzumab emtansine | 21234907 | dm+d
Trastuzumab emtansine | 21179199 | dm+d

## Suggestion:
The above 5 concepts need to be mapped to RxNorm concept of ado-trastuzumab emtansine (43525787).

**Comment by cgreich** [comment]
True. Will fix.

**Comment by ericaVoss** [comment]
Testing in 20180609

I don't have GRR since @cgreich won't let me have it . . . but . . . 

```sql
WITH CTE_DRUGS_OF_INTEREST AS (
	SELECT *
	FROM CONCEPT
	WHERE CONCEPT_ID IN (
	41605044,43243659,21309257,21234907,21179199
	)
)
SELECT i.CONCEPT_ID, i.CONCEPT_NAME, i.VOCABULARY_ID, c.CONCEPT_ID, c.CONCEPT_NAME, c.CONCEPT_CLASS_ID
FROM CTE_DRUGS_OF_INTEREST i
	LEFT OUTER JOIN CONCEPT_RELATIONSHIP cr
		ON cr.CONCEPT_ID_1 = i.CONCEPT_ID
	LEFT OUTER JOIN CONCEPT c
		ON c.CONCEPT_ID = cr.CONCEPT_ID_2
```

Still being mapped to 1387104-trastuzumab

**Comment by cgreich** [comment]
Aren't you guys buying DA GErmany? That's what it is in tehre. You can have it.

**Comment by ericaVoss** [comment]
YES, we have DA Germany!!  But you refuse to approve my permissions to it.  

See #137 

![bitmoji](https://render.bitstrips.com/v2/cpanel/822076c9-6743-49b9-8de3-3fbce68f5f20-f9de47cd-e918-4b50-831d-d887a6386e5f-v1.png?transparent=1&palette=1&width=246)

**Comment by abedtashh** [comment]
I just checked Athena. Not fixed yet!

There is a shortcut to see which concepts need to be changed: All "Trastuzumab emtansine" concepts that are incorrectly mapped to "Trastuzumab" RxNorm concept (here: http://athena.ohdsi.org/search-terms/terms/1387104) need to be mapped to "ado-trastuzumab emtansine" RxNorm concept (http://athena.ohdsi.org/search-terms/terms/43525787).

**Comment by aostropolets** [comment]
Hamed, it’s not that easy as you think. We need to update and refresh all the vocabularies that are involved. It apparently takes time and resources. We will do it as soon as possible. Will keep you posted.

**Comment by deepakyedlarajaiah** [comment]
Please advise if this issue has been fixed.we are encountering the same issue.

**Comment by TinyRickC137** [comment]
@deepakyedlarajaiah it has not been fixed yet - to fix this problem, we need to change the logic of drug processing in OMOP completely. RxNorm is a foundation of a Drug domain in OHDSI, and RxNorm treats trastuzumab emtansine as a [precise ingredient](https://athena.ohdsi.org/search-terms/terms/43525787) (a form of an ingredient and not a distinct concept). There are multiple drugs with the same problem. 

Currently, we work according to the [roadmap](https://github.com/OHDSI/Vocabulary-v5.0/wiki/Release-planning), and the plan is to fix the condition domain first (almost done), then switch to drugs. I will expect changes in the upcoming years, but it is like a change of the whole foundation of the building - not easy at all.
